lib/polyfill/mediasource.js

```diff
@@ -35,19 +35,12 @@ shaka.polyfill.MediaSource = class {
     } else if (safariVersion && window.MediaSource) {
       // NOTE:  shaka.Player.isBrowserSupported() has its own restrictions on
       // Safari version.
-      if (safariVersion <= 10) {
+      if (safariVersion <= 12) {
+        shaka.log.info('Patching Safari 8-12 MSE bugs.');
         // Safari 8 does not implement appendWindowEnd.
-        // Safari 9 & 10 do not correctly implement abort() on SourceBuffer.
-        // Bug filed: https://bugs.webkit.org/show_bug.cgi?id=160316
-        // Blacklist these very outdated versions.
-        // Bug filed: https://bugs.webkit.org/show_bug.cgi?id=165342
         // Safari 10 fires spurious 'updateend' events after endOfStream().
-        // Bug filed: https://bugs.webkit.org/show_bug.cgi?id=165336
+        // We do not have patches for these bugs here.
 
-        shaka.log.info('Blacklisting MSE on Safari <= 10.');
-        shaka.polyfill.MediaSource.blacklist_();
-      } else if (safariVersion <= 12) {
-        shaka.log.info('Patching Safari 11 & 12 MSE bugs.');
         // Safari 11 & 12 do not correctly implement abort() on SourceBuffer.
```

Comment:
```suggestion
        // Safari 8-12 do not correctly implement abort() on SourceBuffer.
```

Also the line 50 please!